docs/imviz/displayimages.rst

```diff
@@ -0,0 +1,85 @@
+.. _imviz-display-images:
+
+*****************
+Displaying Images
+*****************
+
+Imviz uses image viewers to visualize data from supported formats.
+
+Selecting Data Set
+==================
+
+.. seealso::
+
+    :ref:`Selecting Data Set<cubeviz-selecting-data>`
+        Documentation on selecting data sets in the Jdaviz viewers.
+
+
+Adding New Viewers
+==================
+
+In the toolbar towards the top of the UI, there is a :guilabel:`picture with +` icon. By
+clicking this icon, you can add new viewers to the application. You can then select from the data
+that has been loaded into the application to be visualized in these additional viewers.
+You can then utilize some of the Imviz-specific features, like Linked Pan/Zoom.
+
+Defining Spatial Regions
+========================
+
+Spatial regions can be created the same way as in Cubeviz.
+
+.. seealso::
+
+    :ref:`Defining Spatial Regions <spatial-regions>`
+        Documentation on defining spatial regions in an image viewer.
+
+Blinking
+========
+
+Blinking is an Imviz-specific functionality that allows a user to quickly switch
+between viewing two images. This can be done by selecting the :guilabel:`hammer and screwdriver`
+icon and then clicking the :guilabel:`right arrow` icon. You can also access blinking
+by selecting the "b" key on your keyboard.
+
+Contrast/Bias
+=============
+
+In addition to changing contrast and bias information in the :guilabel:`layer` tab
+(see :ref:`Contrast <contrast>`), Imviz has a :guilabel:`white to black` button under the
+:guilabel:`hammer and screwdriver` menu that can also edit those options. After clicking
+that button, you can click and drag on the image viewer to change to change the contrast
+and bias. If you would like to reset to the default contrast and bias settings, you can
```

Comment:
Which direction changes which property?

Reply:
I think that's a question for @pllim , I'd have to spend some time figuring it out.

Reply:
https://github.com/spacetelescope/jdaviz/blob/79e4245663691a5f6567993be4c0198edebf4fbb/jdaviz/configs/imviz/plugins/tools.py#L150-L152

Drag across X changes bias, Y contrast. 😸